web/src/beta/features/PluginPlayground/Plugins/index.tsx

```diff
@@ -127,24 +127,32 @@ const Plugins: FC<Props> = ({
   return (
     <Wrapper>
       <IconList>
-        <IconButton
-          appearance="simple"
+        <Button
```

Comment:
It seems to be a typical icon button, therefore i prefer keep using `IconButton`. Ref:
```
        <IconButton
          icon="addFile"
          size="large"
          hasBorder
          tooltipText={t("Add File")}
          placement="top"
          onClick={() => setIsAddingNewFile(true)}
        />
```

Reply:
Okay, it was previously iconButton, however, iconButton does not have same style as the design, which is why i changed it to Button (the same with the run code icon), if i return it to iconButton, it wont have the styling on it anymore.

Reply:
Understood, we should communicate with designers around this. For now let's keep them as Buttons.